spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1736,7 +1736,7 @@ def setup_firefox_with_linux
           end
           context "when with snapshots" do
             before { allow(@record).to receive_message_chain(:snapshots, :size).and_return(2) }
-            it_behaves_like 'default case'
+            it_behaves_like 'record with error message', 'revert_to_snapshot'
```

Comment:
For my educational purposes: whats the exact difference between those two shared examples? If I remember correct the 'default case' tests the method returns false, right?